frontend/packages/operator-lifecycle-manager/src/dev-catalog.ts

```diff
@@ -33,7 +33,9 @@ export const normalizeClusterServiceVersions = (
     `${crdDescription}\n## Operator Description\n${csvDescription}`;
 
   const operatorProvidedAPIs: K8sResourceKind[] = _.flatten(
-    clusterServiceVersions.map((csv) => providedAPIsFor(csv).map((desc) => ({ ...desc, csv }))),
+    clusterServiceVersions.map((csv) => {
+      return providedAPIsFor(csv).map((desc) => ({ ...desc, csv }));
```

Comment:
This change is unnecessary as you do not need a return statement for a single line function.

Reply:
Yeah.... I console logged the desc and forgot to remove the {}.. will remove this